### PR TITLE
Added placeholder for tutorials at engo.io

### PR DESCRIPTION
**Pull request by EtienneBruines**
Actual tutorials may be added, but not in this PR. 

Would anyone please review this? :-)

(Simply checkout the `tutorials` branch, enter `jekyll serve`, and visit `http://localhost:4000`)

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -2,11 +2,9 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
 	
-	<title>Engo</title>
 	<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/foundation/6.2.0/foundation.min.css">
 	<style type="text/css">
 		body { text-align: center; }
 		.row { max-width: 35rem; }
 		footer p { font-size: small; color: gray; }
 	</style>
-</head>
```
Comment (paked) [inline _includes/head.html:12]: I don't think we should break the markup here.

[A better solution would be to use Jekyll variables](https://jekyllrb.com/docs/variables/)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (paked) [inline index.html:1]: I'd prefer our site be written in markdown if at all possible.

FYI you can use some HTML inside of the markdown, but it looks like everything you use here is supported natively.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+layout: default
+title: Welcome
+permalink: /
+---
+
+<h2>Welcome</h2>
```
Comment (paked) [inline index.html]: Because you have declared the `title` variable in the frontmatter, you can do {{ title }}. However, it'd be best to put this in the head include IMO.

File: tutorials/_posts/2016-04-04-01-hello-world.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (paked) [inline tutorials/_posts/2016-04-04-01-hello-world.html]: This is a default page, we can remove it right?

File: tutorials/_posts/2016-04-04-02-custom-systems.html [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+layout: tutorial
```
Comment (paked) [inline tutorials/_posts/2016-04-04-02-custom-systems.html]: Lets try and get this written in Markdown

**Comment by paked** [comment]
Made comments.

TLDR: Markdownify and use frontmatter

File: index.html (hunk 2 above)
Comment (EtienneBruines) [inline index.html:1]: The reason I'm using HTML is because I like the idea of having two things beside each other (and I have no idea how to do that in Markdown). Don't know if it's possible in markdown. 

I also used templating/etc. by Foundation Zurb (simply add the css file, and special classes for the markup), and it automatically makes it look nice on each screen size (from mobile to 4K and everything in between). Not sure if this is compatible with markdown / if markdown does the same?

Comment (paked) [inline index.html:1]: Ah OK. I understand now. Sorry for misinterpreting the situation.

File: index.html (hunk 3 above)
Comment (paked) [inline index.html]: I also don't think we need the permalink here

File: index.html (hunk 2 above)
Comment (EtienneBruines) [inline index.html:1]: I have nothing against markdown / am fine with using markdown, I'm just not as flexible with it as with HTML. 

It's up to you :-). (the age-old dilemma between flexibility and simplicity)

File: index.html (hunk 3 above)
Comment (EtienneBruines) [inline index.html]: I think you're right :+1:

File: tutorials/_posts/2016-04-04-01-hello-world.html (hunk 4 above)
Comment (EtienneBruines) [inline tutorials/_posts/2016-04-04-01-hello-world.html]: Remove what?

Comment (paked) [inline tutorials/_posts/2016-04-04-01-hello-world.html]: Sorry. I thought it was something else :dancer:. As you were ;)
